### ISSUE TITLE
3DS 2.0 instance twice

### ISSUE THREAD

**Issue by FrancescoBarrile** [braintree/braintree-android-drop-in#213]
General information

-   SDK/Library version: 5.1.0
-   Environment: Production
-   Android Version and Device: ALL

Issue description

We have an issue on our app with Android os, when the app opens the ACS page, on the phone we see two instances: one of our app and one of the ACS web page, but we only expect our app instance.
Many users, after making the authorization, return to our app and not to ACS page, this causes the operation to stop.
We declared the browser switch in the manifest:

```
<activity android:name="com.braintreepayments.api.BraintreeBrowserSwitchActivity"
    android:launchMode="singleTask">
    <intent-filter>
        <action android:name="android.intent.action.VIEW" />
        <category android:name="android.intent.category.DEFAULT" />
        <category android:name="android.intent.category.BROWSABLE" />
        <data android:scheme="com.goelectricstations.nextcharge.braintree" />
    </intent-filter>
</activity>
```

attached a screenshot to show the issue.

![image](https://user-images.githubusercontent.com/66623992/107776209-911f9500-6d41-11eb-9898-abc0aaa43b32.png)

**Comment by sarahkoop** [comment]
Hi @FrancescoBarrile - This is currently the expected browser switching behavior. We are aware of the issue when a user navigates back to the app in the middle of the authorization process. We have released a [beta version ](https://github.com/braintree/browser-switch-android/releases/tag/2.0.0-beta1)of the underlying `browser-switch-android` library that should resolve this issue. We will post an update here when that change is available in Braintree Android Drop In.

**Comment by FrancescoBarrile** [comment]
@sarahkoop thank you

**Comment by sarahkoop** [comment]
Hi @FrancescoBarrile we've released version `6.0.0-beta2` of the drop-in SDK that uses the upgraded `browser-switch-android` library and should resolve this issue.

**Comment by sarahkoop** [comment]
Version `6.0.0` of the Drop-in SDK has been released with this fix.
